<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
border:2px groove gray;
border-radius:12px;
padding:2px 4px;
	background-image: url(:/image/image/login1.jpg);
color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>34</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 34pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>    文渊阁</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_account">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>331</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>331</width>
      <height>61</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
font: 14pt &quot;楷体&quot;;
padding:2px 4px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(153,195,211,255), stop:1 rgba(152,195,212, 0));</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>11</number>
    </property>
    <property name="placeholderText">
     <string>读者证号/管理员账号</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_password">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>331</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>331</width>
      <height>61</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
font: 14pt &quot;楷体&quot;;
padding:2px 4px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(153,195,211,255), stop:1 rgba(152,195,212, 0));</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_close">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_close{
background-image: url(:/image/image/icon/close.png);
border-radius:2px;
padding:2px 4px;
color: rgb(0,128,128);
background-color: rgba(255,255,255, 96);
}

QPushButton#button_close:hover{
background-image: url(:/image/image/icon/close_hover.png);
border-radius:2px;
padding:2px 4px;
color: rgb(0,128,128);
background-color: rgba(240,255,255,144);
}
 
QPushButton#button_close:pressed{
background-image: url(:/image/image/icon/close_press.png);
border-radius:2px;
padding:2px 4px;
color: rgb(0,128,128);
background-color: rgba(225,255,255, 156);
padding-left:6px;
padding-top:6px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_login">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_login{
	font: 16pt &quot;楷体&quot;;
    color: rgb(255, 255, 255);
    padding-top:-6px;

	border-image: url(:/image/image/anniu.png);
}
QPushButton:hover#button_login{
	font: 16pt &quot;楷体&quot;;;
    color: rgb(255, 255, 255);
padding-top:-6px;

	border-image: url(:/image/image/anniu2.png);
}
QPushButton:pressed#button_login{
	font: 16pt &quot;楷体&quot;;
    color: rgb(255, 255, 255);
padding-top:-6px;

	border-image: url(:/image/image/anniu2.png);
}</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_mini">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_mini{
background-image: url(:/image/image/icon/minimize.png);
border-radius:2px;
padding:2px 4px;
color: rgb(0,128,128);
background-color: rgba(255,255,255, 96);
}

QPushButton#button_mini:hover{
background-image: url(:/image/image/icon/minimize_hover.png);
border-radius:2px;
padding:2px 4px;
color: rgb(0,128,128);
background-color: rgba(240,255,255,144);
}
 
QPushButton#button_mini:pressed{
background-image: url(:/image/image/icon/minimize_press.png);
border-radius:2px;
padding:2px 4px;
color: rgb(0,128,128);
background-color: rgba(225,255,255, 156);
padding-left:6px;
padding-top:6px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_admin">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
    </property>
    <property name="text">
     <string>管理员</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_reader">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
    </property>
    <property name="text">
     <string>读者</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
